packages/client/src/views/GrantDetailsView.vue

```diff
@@ -29,6 +29,65 @@
           {{ currentGrant.title }}
         </h2>
 
+        <!-- Left page column: main print/copy/grants.gov buttons -->
+        <div class="grant-details-main-actions print-d-none">
+          <div class="d-flex justify-content-between align-items-center">
+            <b-button
+              variant="primary"
+              block
+              class="mr-5"
+              style="width: fit-content; white-space: nowrap; "
+              :href="`https://www.grants.gov/search-results-detail/${currentGrant.grant_id}`"
+              target="_blank"
+              rel="noopener noreferrer"
+              data-dd-action-name="view on grants.gov"
+              @click="onOpenGrantsGov"
+            >
+              <b-icon
+                icon="box-arrow-up-right"
+                aria-hidden="true"
+                class="mr-2"
+              />
+              View on Grants.gov
+            </b-button>
+            <div class="w-20 d-flex">
+              <a
+                class="link-primary"
+                role="button"
+                :variant="copyUrlSuccessTimeout === null ? 'outline-primary' : 'outline-success'"
+                data-dd-action-name="copy btn"
+                style="white-space: nowrap; "
+                @click="copyUrl"
+              >
+                <b-icon
+                  :icon="copyUrlSuccessTimeout === null ? 'paperclip' : 'check2'"
+                  aria-hidden="true"
+                  class="mr-2"
+                />
+                <span v-if="copyUrlSuccessTimeout === null">Copy Link</span>
+                <span v-else>Link Copied</span>
+              </a>
+              <div class="col-1 border-right border-dark p-2"/>
```

Comment:
Seeing some linting errors here. Ideally you shouldn't have to think about style stuff like this, and you can get your editor set up to auto-format on save. A bit of documentation on getting set up is here — https://github.com/usdigitalresponse/usdr-gost/blob/main/docs/development.md#linting — but lmk if you have any trouble getting that working!